parlai/crowdsourcing/tasks/acute_eval/README.md

```diff
@@ -0,0 +1,101 @@
+# ACUTE-Eval
+
+## Paper information
+
+Margaret Li, Jason Weston, Stephen Roller.
+_[ACUTE-EVAL: Improved Dialogue Evaluation with Optimized Questions and Multi-turn Comparisons](https://arxiv.org/abs/1909.03087)_.
+
+## Citation
+
+If you use this evaluation method in your own work, please cite with the
+following BibTex entry:
+
+    @misc{li2019acuteeval,
+      title={ACUTE-EVAL: Improved Dialogue Evaluation with Optimized Questions and Multi-turn Comparisons},
+      author={Margaret Li and Jason Weston and Stephen Roller},
+      year={2019},
+      journal={Advances in Neural Information Processing Systems, Conversational AI Workshop},
+      url={https://arxiv.org/abs/1909.03087}
+    }
+
+# Code Instructions
+Once you have installed [ParlAI](https://github.com/facebookresearch/ParlAI/#installing-parlai) and [Mephisto](https://github.com/facebookresearch/mephisto/blob/master/docs/quickstart.md), follow the instructions below.
+
+The `run.py` script is designed to allow you to run this entire task from command line with an invocation like
```

Comment:
It may make sense to rename `example_script.py`, or update this comment.

Reply:
Whoops - yes, changing